keras/utils/training_utils.py

```diff
@@ -95,14 +95,24 @@ def multi_gpu_model(model, gpus):
     if K.backend() != 'tensorflow':
         raise ValueError('`multi_gpu_model` is only available '
                          'with the TensorFlow backend.')
-    if gpus <= 1:
-        raise ValueError('For multi-gpu usage to be effective, '
-                         'call `multi_gpu_model` with `gpus >= 2`. '
-                         'Received: `gpus=%d`' % gpus)
+    if isinstance(gpus, int):
+        if gpus <= 1:
+            raise ValueError('For multi-gpu usage to be effective, '
+                             'call `multi_gpu_model` with `gpus >= 2`. '
+                             'Received: `gpus=%d`' % gpus)
+        n_gpu = gpus
+        target_gpu_id = list(range(n_gpu))
+    elif isinstance(gpus, list):
+        if len(gpus) <= 1:
+            raise ValueError('For multi-gpu usage to be effective, '
+                             'call `multi_gpu_model` with `len(gpus) >= 2`. '
+                             'Received: `gpus=%d`' % gpus)
```

Comment:
Use `%s` since this will be a list (or tuple, soon)

Reply:
Additionally, the `ValueError` should be tested (there's already a test for the other ones); this will allow you to spot this kind of issue